### DISCUSSION TITLE
AI is not working

### DISCUSSION THREAD

**Discussion by gregiteen** [webstudio-is/webstudio-community#141] [Q&A] [answered]
I tried all three of the suggested prompts and got errors:  Not valid json, and too many tokens

**Comment by gregiteen** [comment]
![image](https://github.com/webstudio-is/webstudio-community/assets/155279229/0966dabb-2ef9-4ff2-8492-217989b597b0)

**Reply by kof** [reply, marked as answer]
The error message is the result of the model not being able to process too much context. Amount of context depends on the selected instance. We need a better UX for this.

**Reply by ovsw** [reply]
The only prompt that worked out of the example ones was the first one (image with mountains, blurred) which looked quite bad (literally just a blurred image). The rest all failed with the attached error, saying the context was not big enough.
![CleanShot 2025-03-10 at 15 00 01@2x](https://github.com/user-attachments/assets/745c7acc-92d1-4f42-aa3c-f810fa49c5e2)
![CleanShot 2025-03-10 at 15 00 21@2x](https://github.com/user-attachments/assets/a45c0b29-dd16-48a9-8854-136256fa5627)
![CleanShot 2025-03-10 at 14 56 25@2x](https://github.com/user-attachments/assets/d7a0501b-7082-4991-9fec-545f33cdd557)

**Reply by kof** [reply]
I assume you selected a something on the page that has a lot of content inside?

**Reply by kof** [reply]
We will be switching to newer/better models soon.
